### DISCUSSION TITLE
How to set `maxFeeds` in `actualize_script.php` for systemd timer?

### DISCUSSION THREAD

**Discussion by amalgame21** [FreshRSS/FreshRSS#6550] [Q&A]
My low spec server cannot run the `actualize_script.php` script while playing video.
How can I lower the CPU consumption of the script, like reduce the number of feed updatedsimutaneously?
Thanks.

**Comment by math-GH** [comment]
You can configure it

The constant's name is `nb_parallel_refresh`.

How to configure the `constants.php`: https://freshrss.github.io/FreshRSS/en/admins/17_configs_not_ui.html

Does it help?

**Reply by Alkarex** [reply]
`nb_parallel_refresh` is only for manual refresh from the Web UI. It is unrelated to cron jobs running automatically server-side. At the moment, cron jobs are refreshing one feed at a time, and the CPU consumption should remain relatively low. It would be interesting to have a screenshot of the `htop` or similar tools to see what is taking CPU time.
